content/zh-cn/releases/download.md

```diff
@@ -91,47 +125,7 @@ To manually verify signed container images of Kubernetes core components, refer
 
 <!--
 ## Binaries
-
-Find links to download Kubernetes components (and their checksums) in the
-[CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG) files.
-
-Alternately, use [downloadkubernetes.com](https://www.downloadkubernetes.com/) to filter by version and architecture.
 -->
 ## 二进制  {#binaries}
 
-在 [CHANGELOG](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG) 文件中找到下载 Kubernetes 组件（及其校验和）的链接。
-
-或者，使用 [downloadkubernetes.com](https://www.downloadkubernetes.com/) 按版本和架构进行过滤。
-
-### kubectl
-
-<!-- overview -->
-
-<!--
-The Kubernetes command-line tool, [kubectl](/docs/reference/kubectl/kubectl/), allows
-you to run commands against Kubernetes clusters.
-
-You can use kubectl to deploy applications, inspect and manage cluster resources,
-and view logs. For more information including a complete list of kubectl operations, see the
-[`kubectl` reference documentation](/docs/reference/kubectl/).
--->
-Kubernetes 命令行工具 [kubectl](/zh-cn/docs/reference/kubectl/kubectl/) 允许你对 Kubernetes 集群执行命令。
-
-你可以使用 kubectl 部署应用程序、检查和管理集群资源以及查看日志。
-有关更多信息，包括 kubectl 操作的完整列表，请参阅
-[`kubectl` 参考文档](/zh-cn/docs/reference/kubectl/)。
-
-<!--
-kubectl is installable on a variety of Linux platforms, macOS and Windows.
-Find your preferred operating system below.
-
-- [Install kubectl on Linux](/docs/tasks/tools/install-kubectl-linux)
-- [Install kubectl on macOS](/docs/tasks/tools/install-kubectl-macos)
-- [Install kubectl on Windows](/docs/tasks/tools/install-kubectl-windows)
--->
-kubectl 可安装在各种 Linux 平台、macOS 和 Windows 上。
-在下方找到你首选的操作系统。
-
-- [在 Linux 上安装 kubectl](/zh-cn/docs/tasks/tools/install-kubectl-linux)
-- [在 macOS 上安装 kubectl](/zh-cn/docs/tasks/tools/install-kubectl-macos)
-- [在 Windows 上安装 kubectl](/zh-cn/docs/tasks/tools/install-kubectl-windows)
+{{< release-binaries >}}
```

Comment:
A brief heads-up: Currently with these changes there are some section of this page ([preview](https://deploy-preview-43933--kubernetes-io-main-staging.netlify.app/zh-cn/releases/download/#binaries)) that are partially rendering in English. The `release-binaries` shortcode utilizes localized strings in [en.toml,](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml) and adding the corresponding entries in [zh-ch.toml ](https://github.com/kubernetes/website/blob/main/data/i18n/zh-cn/zh-cn.toml)would ensure a proper reflection in the localized version.

Check out commit [0c514f8](https://github.com/kubernetes/website/commit/0c514f8e58376002cd43d1561ab0cc94f947a85e#diff-ad798e4860c4fe33189e78558f8eebb4ed6fc380293e6626744369453edca684) for the list of strings added in en.toml for this shortcode. Alternatively, syncing en.toml and zh-ch.toml entirely is another option.

Reply:
Thanks, will update zh-cn.toml soon.